bin/deb_local

```diff
@@ -67,7 +68,7 @@ def build_locally(packager, pkg_name, options)
             raise RuntimeError, "Local rebuild of pkg #{pkg_name} failed -- see #{rebuild_log}"
         end
         puts "############### build #{pkg_name} #####################"
-        debian_package_dir = packager.packaging_dir(options[:debian_package_name])
+        debian_package_dir = packager.packaging_dir_i(options[:debian_package_name])
```

Comment:
packaging_dir also allows a PackageInfo description. The conversion _i to _info and pkgi to pkginfo is easily done. I was wondering if it is better to just drop the suffix? Would decrease the size of the diff substantially, but removes the information where different objects are passed around.